internal/jobobject/jobobject.go

```diff
@@ -430,18 +430,13 @@ func (job *JobObject) QueryProcessorStats() (*winapi.JOBOBJECT_BASIC_ACCOUNTING_
 }
 
 // QueryStorageStats gets the storage (I/O) stats for the job object.
-func (job *JobObject) QueryStorageStats() (*winapi.JOBOBJECT_BASIC_AND_IO_ACCOUNTING_INFORMATION, error) {
-	job.handleLock.RLock()
```

Comment:
Do we not need to lock on the handle anymore?

Reply:
Oh crap, good catch. We do, just got lost in the refactor